panacea/grpc_client.go

```diff
@@ -52,7 +55,29 @@ type GrpcClient struct {
 
 func NewGrpcClient(conf *config.Config) (GrpcClientI, error) {
 	log.Infof("dialing to Panacea gRPC endpoint: %s", conf.Panacea.GRPCAddr)
-	conn, err := grpc.Dial(conf.Panacea.GRPCAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+
+	var conn *grpc.ClientConn
+	var err error
+
+	u, err := url.Parse(conf.Panacea.GRPCAddr)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse grpc addr: %w", err)
+	}
+
+	prefixLen := len(u.Scheme + "://")
+	addrBody := conf.Panacea.GRPCAddr[prefixLen:]
+
+	var creds credentials.TransportCredentials
+
+	if u.Scheme == "tcp" || u.Scheme == "http" {
```

Comment:
When I test, if shecma(http, https...) is missing, the schema responds with an empty value.
I think it would be better to either receive the schema as mandatory or add http as the default.

Reply:
Let's make it mandatory. Let's return an error if the user doesn't specify the scheme.